docs(*) fix versions in links TD-12 (#1130)
Kong/docs.konghq.com: app/enterprise/0.32-x/kong-implementation-checklist.md

These are the port settings in order for Kong to work: 

- Allow HTTP traffic to [proxy_listen](/0.13.x/configuration/#proxy_listen). By default  8000. 

> - Allow HTTPS traffic to [proxy_listen_ssl](/0.13.x/configuration/#proxy_listen_ssl). By default 8443.

Reviewer: This jumps to the top of the page. There's no anchor link.

Author: There may be a separate issue with anchor links OSS of this PR. Hard link works correctly: https://docs.konghq.com/0.13.x/configuration/#proxy_listen

Author: Ah, you were referring to /#proxy_listen_ssl. It seems this property is entirely absent. Will check with Core.

Author: ```suggestion
```

- Allow HTTP traffic to [admin_listen](/0.13.x/configuration/#admin_listen). By default 8001.

> - Allow HTTPS traffic to [admin_listen_ssl](/0.13.x/configuration/#admin_listen_ssl). By default 8444.

Author: ```suggestion
```

- Allow HTTP traffic to admin_gui_listen. By default 8002.
- Allow HTTPS traffic to admin_gui_listen_ssl. By default 8445.

### Firewall

Below are the recommended firewall settings:

> - The upstream APIs behind Kong will be available on [proxy_listen](/0.13.x/configuration/#proxy_listen) and [proxy_listen_ssl](/0.13.x/configuration/#proxy_listen_ssl). Configure these ports accordingly to the access level you wish to grant to the upstream APIs.

Author: ```suggestion
- The upstream APIs behind Kong will be available on [proxy_listen](/0.13.x/configuration/#proxy_listen). Configure this port accordingly to the access level you wish to grant to the upstream APIs.
```
